demos/CMakeLists.txt

```diff
@@ -9,7 +9,6 @@ afr_module_sources(
         "$<$<TARGET_EXISTS:AFR::freertos_plus_tcp>:${AFR_DEMOS_DIR}/demo_runner/aws_demo_network_addr.c>"
         "${AFR_DEMOS_DIR}/demo_runner/aws_demo.c"
         "${AFR_DEMOS_DIR}/network_manager/aws_iot_network_manager.c"
-        "${AFR_DEMOS_DIR}/network_manager/aws_iot_demo_network.c"
```

Comment:
Could you also remove header file `demos/include/aws_iot_demo_network.h`

Reply:
Thanks for pointing it out, removed.